nservicebus/hosting/azure-functions-service-bus/index.md

```diff
@@ -65,6 +65,8 @@ For recoverability to move the continuously failing messages to the error queue
 
 snippet: asb-configure-error-queue
 
+partial: assembly-scanner
```

Comment:
I'd vote to put this at the complete end of the documentation.

Reply:
sounds good. I'll move it.